web_export_view/static/src/js/web_export_view.js

```diff
@@ -50,14 +50,16 @@ odoo.define('web_export_view', function (require) {
             var export_columns_names = [];
             var column_index = 0;
```

Comment:
Wouldn't it be the same to just put a `1` here? :thinking:

Reply:
Nope :/ The problem is "export_columns_keys"... if you set to 1 in non-grouped views you get an exception... and in grouped you see good headers but bad column content.
This fix really is a "workaround". For me a best solution would be to improve "ListRenderer" and store header strings, avoiding css query selector usage.

Reply:
This will turn it `1` when it is grouped, and `0` otherwise:

```suggestion
             // Skip 1st empty header cell if list is grouped
             var column_index = view.renderer.state.groupedBy.length && 1;
```